scripts/python/compute_retrieval_map.py

```diff
@@ -0,0 +1,107 @@
+import os
```

Comment:
Let's move this script to the vote-and-verify repository?

Reply:
Works for me. It will probably be better noticed if we put it there, anyhow.